libs/application/template-api-modules/src/lib/modules/templates/driving-license-submission/driving-license-submission.service.ts

```diff
@@ -146,11 +146,12 @@ export class DrivingLicenseSubmissionService extends BaseTemplateApiService {
     const needsHealthCert = calculateNeedsHealthCert(answers.healthDeclaration)
     const remarks = answers.hasHealthRemarks === 'yes'
     const needsQualityPhoto = answers.willBringQualityPhoto === 'yes'
-    const jurisdictionId = answers.jurisdiction
+    const jurisdictionId = Number(getValueViaPath(answers, 'delivery.jurisdiction'))
```

Comment:
Future todo: 
It seems we're only doing boolean comparisons with this and then defaulting to 37 on false.
Maybe we should make sure that there isn't a jurisdiction with the number 0 since it would default to Kópavogur.
Since Boolean(Number(0)) is false